### PR TITLE
Update qandidate/toggle dependency to support versions 1.* < 2.0.0

### PR DESCRIPTION
**Pull request by fatganz**
Solves #25

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,10 +31,5 @@
     },
     "suggest": {
         "twig/twig": "For using the twig helper"
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "0.5.x-dev"
-        }
```
Comment (Spea) [inline composer.json]: This should not be removed

Comment (fatganz) [inline composer.json]: Sorry. Not sure how this went into PR. It was local commit only.

**Comment by wjzijderveld** [comment]
This bundle doesn't _require_ >1.0, it still works fine with < 1.0. So it might be better to do something like this;

`^0.1|^1.0`, That means it also allows both >=0.1 and >=1.0?

**Comment by fatganz** [comment]
@wjzijderveld yeah, you're right. And seems like this request doesn't make any sense after merging SF3 compatibility PR

**Pull request closed without merging**